### PR TITLE
WIP OADP-4610: Fix name keypair value oadp dpa

### PR DESCRIPTION
**Pull request by anarnold97**
### Jira

* [OADP-4610](https://issues.redhat.com/browse/OADP-4610)

Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by shdeshpa07** [comment]
@anarnold97 - LGTM

**Comment by shdeshpa07** [comment]
I looked at the DPA 1.2 and 1.3 for 4.14, 4.15 and 4.16 and found 'name' key value pair at the following locations:

**4.14:**

DPA 1.2:
name key-value pair in: aws bsl, azure vsl

DPA 1.3:
name key-value pair in: aws bsl and vsl, azure vsl


**4.15:**

DPA 1.2:
name key-value pair in: aws bsl, azure vsl

DPA 1.3:
name key-value pair in: aws bsl and vsl, azure vsl


**4.16:**

DPA 1.2:
name key-value pair in: aws bsl, azure vsl

DPA 1.3:
name key-value pair in: aws bsl and vsl, azure vsl

File: modules/oadp-installing-dpa-1-3.adoc [hunk 1]
```diff
@@ -65,8 +65,7 @@ spec:
       podConfig:
         nodeSelector: <node_selector> # <7>
   backupLocations:
-    - name: default
-      velero:
+    - velero:
```
Comment (anarnold97) [inline modules/oadp-installing-dpa-1-3.adoc:68]:
```suggestion
    - name: default
      velero:
```

File: modules/oadp-installing-dpa-1-2-and-earlier.adoc [hunk 2]
```diff
@@ -69,8 +69,7 @@ spec:
       podConfig:
         nodeSelector: <node_selector> # <4>
   backupLocations:
-    - name: default
-      velero:
+    - velero:
```
Comment (anarnold97) [inline modules/oadp-installing-dpa-1-2-and-earlier.adoc:72]:
```suggestion
    - name: default
      velero:
```

**Comment by anarnold97** [comment]
> I looked at the DPA 1.2 and 1.3 for 4.14, 4.15 and 4.16 and found 'name' key value pair at the following locations:
> 
> **4.14:**
> 
> DPA 1.2: name key-value pair in: aws bsl, azure vsl
> 
> DPA 1.3: name key-value pair in: aws bsl and vsl, azure vsl
> 
> **4.15:**
> 
> DPA 1.2: name key-value pair in: aws bsl, azure vsl
> 
> DPA 1.3: name key-value pair in: aws bsl and vsl, azure vsl
> 
> **4.16:**
> 
> DPA 1.2: name key-value pair in: aws bsl, azure vsl
> 
> DPA 1.3: name key-value pair in: aws bsl and vsl, azure vsl

Please can you give me line numbers?

**Comment by shdeshpa07** [comment]
> > I looked at the DPA 1.2 and 1.3 for 4.14, 4.15 and 4.16 and found 'name' key value pair at the following locations:
> > **4.14:**
> > DPA 1.2: name key-value pair in: aws bsl, azure vsl
> > DPA 1.3: name key-value pair in: aws bsl and vsl, azure vsl
> > **4.15:**
> > DPA 1.2: name key-value pair in: aws bsl, azure vsl
> > DPA 1.3: name key-value pair in: aws bsl and vsl, azure vsl
> > **4.16:**
> > DPA 1.2: name key-value pair in: aws bsl, azure vsl
> > DPA 1.3: name key-value pair in: aws bsl and vsl, azure vsl
> 
> Please can you give me line numbers?

Here are the line numbers for VSL only:

**4.14:**

DPA 1.2: azure vsl line no. 152

DPA 1.3: aws vsl line 84, azure vsl line 153


**4.15:**

DPA 1.2: azure vsl line 152

DPA 1.3: aws vsl line 84, azure vsl line 153


**4.16:**

DPA 1.2: azure vsl line 152

DPA 1.3: aws vsl line 84, azure vsl line 153

**Comment by anarnold97** [comment]
@shdeshpa07 - please can you peer-review?
Thanks for the line numbers think we have it

**Comment by shdeshpa07** [comment]
@anarnold97 - LGTM. Thanks.

**Comment by anarnold97** [comment]
@PrasadJoshi12 - please could you review this PR to fix the VSL `name`?

Thanks

**Pull request closed without merging**